Dockerfile

```
@@ -18,8 +18,29 @@ WORKDIR /home/k6
 ENTRYPOINT ["k6"]
 
 # Legacy loadimpact/k6 image
-FROM release
+FROM release as legacy
 
 COPY entrypoint-legacy.sh /usr/bin/
 
 ENTRYPOINT ["/usr/bin/entrypoint-legacy.sh"]
+
+# Browser-enabled bundle
+FROM release as with-browser
+
+USER root
+
+COPY --from=release /usr/bin/k6 /usr/bin/k6
+RUN apk --no-cache add chromium-swiftshader
+
+USER k6
+
+ENV CHROME_BIN=/usr/bin/chromium-browser
+ENV CHROME_PATH=/usr/lib/chromium/
+
+ENV K6_BROWSER_ENABLED=true
+ENV K6_BROWSER_HEADLESS=true
+# Disable RAM-heavy GPU rasterizer and make chromium use /tmp instead of /dev/shm that is typically
+# too small in Docker
+ENV K6_BROWSER_ARGS="--disable-software-rasterizer --disable-dev-shm-usage"
```

I believe this should not include the starting `--`, and should be a comma separated list in our current [implementation](https://github.com/grafana/xk6-browser/blob/1d75cd5b32feba6135a615e6f86bcd1d06bf0fa0/common/browser_options.go#L96). See launch options args [doc](https://k6.io/docs/javascript-api/k6-experimental/browser/browsertype/launch/).
```suggestion
ENV K6_BROWSER_ARGS="disable-software-rasterizer,disable-dev-shm-usage"
```